data/strings/strings.txt

```diff
@@ -19223,7 +19265,7 @@
     ko = %d 추적
     mr = %d ट्रॅक
     nb = %d stier
-    nl = %d routes
+    nl = %d tracks
```

Comment:
In other places "routes" is used for nl. Why do you think that it should be changed to tracks?

Reply:
May be I misunderstood the issue description. Could you please clarify?

https://github.com/organicmaps/organicmaps/issues/4218
![image](https://user-images.githubusercontent.com/126597376/230467219-b2854250-59cf-434f-8694-905a31c2b60b.png)